src/lib/types/experimental.ts

```diff
@@ -299,6 +300,10 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_RESOURCE_LIMITS,
         false,
     ),
+    tokenWidcardOrphans: parseEnvVarBoolean(
```

Comment:
I think `allowOrphanedWildcardTokens` was a really good name. Personally I think you should stick with that. No harm in having a long name I think and the clarity it brings is always nice